ds-collector/ds-collector

```diff
@@ -260,10 +260,11 @@ bastion_checks() {
 }
 
 node_checks() {
+  command -v timeout >/dev/null 2>&1 || { echo >&2 "timeout needs to be installed"; exit 1; }
   [[ -f "$baseDir/$prometheus" ]] || { echo >&2 "$baseDir/$prometheus must exist"; exit 1; }
   [[ -x "$baseDir/$dstat" ]] || { echo >&2 "$baseDir/$dstat must exist"; exit 1; }
   [[ -x "$baseDir/collect-info" ]] || { echo >&2 "$baseDir/collect-info must exist"; exit 1; }
-  [ $(df --output=avail "$baseDir"  | grep -v Avail) -ge 1000000 ] ||  { echo >&2 "There must be at least 1GB free at $baseDir"; exit 1; }
+  [ $(df "$baseDir" | grep -v Filesystem | awk '{print $4}') -ge 1000000 ] ||  { echo >&2 "There must be at least 1GB free at $baseDir"; exit 1; }
```

Comment:
don't use `--output=avail` as it is not just macos that doesn't have this option (older linux is missing it too)